test/integration/multiple-done.spec.js

```diff
@@ -97,7 +97,7 @@ describe('multiple calls to done()', function() {
       expect(res.failures[0], 'to satisfy', {
         fullTitle: 'suite "before all" hook in "suite"',
         err: {
-          message: /done\(\) called multiple times in hook <suite "before all" hook> of file.+multiple-done-before\.fixture\.js/
+          message: /done\(\) called multiple times in hook <suite "before all" hook in "suite"> of file.+multiple-done-before\.fixture\.js/
```

Comment:
this doesn't read quite right... `suite "before all" hook in "suite"`

Reply:
actually, it appears `"suite"` is the _name_ of the suite, which is confusing, so maybe it's not a blocker at all.

Reply:
Yeah, tripped me up, too.